mypyc/analysis/dataflow.py

```diff
@@ -381,6 +400,9 @@ def visit_register_op(self, op: RegisterOp) -> GenAndKill:
     def visit_assign(self, op: Assign) -> GenAndKill:
         return set(op.sources()), {op.dest}
 
+    def visit_set_mem(self, op: SetMem) -> GenAndKill:
+        return set(op.sources()), {op.dest}
```

Comment:
`op.dest` is a pointer so we don't do anything to it, just the memory pointed to by it. Since we don't track properties of memory locations, the second item of the return value should be empty.

Reply:
This should be `set(op.sources()), set()`, since is this essentially same as a void register op.